moveit_experimental/jog_arm/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
```

Comment:
Presumably this will be moved out of moveit_experimental eventually. When I first started, there were other projects inside moveit_experimental that had their own package.xml, so it made sense.